modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,704 @@
+
+When the alternative text exceeds 100 characters, an *Image alternative text should be less than 100 characters* warning presents.
+
+This dialog also presents the alternative text in an editable field, for immediate repair.
+
+==== Replaces the single warning icon presented in dialogs with icons specific to each alert level;
+
+Previously, all **Accessibility Checker** warning dialogs presented with a triangle warning icon, regardless of whether the alert was informational, a warning, or an error.
+
+With this update, **Accessibility Checker** shows a level-specific icon with each alert.
+
+==== The discoverability of the question mark button in warning dialogs was improved
+
+Many **Accessibility Checker** dialogs include a question mark icon. This icon is a link to further information about the discovered issue.
+
+This update includes changes to the presentation of this icon to make discoverability as a clickable button more apparent.
+
+==== The legibility of warning dialogs was improved
+
+In concert with the changes noted above to the question mark icon in **Accessibility Checker** dialogs, legibility improvements were made to the presentation and typography of these dialogs.
+
+The icon’s text was centered and the *Issue*, *Warning*, or *Error* sub-heading’s font size was reduced.
+
+Additionally borders were removed from the exterior of the description field and contrast colors were adjusted to match the background color of the icons added for this release.
+
+==== Added correct icons in dialog alert panel for each severity level
+
+In {productname} 6.3, improvements for the accessibility checker were added to provide additional support for users when trying to identify different alerts such as _info_, _warning_ or _error_ icons.
+
+In previous versions of {productname} the accessibility checker used the same triangle warning icon for all alerts when displayed. In {productname} 6.3, each alert type now displays a unique icon.
+
+* Warning alerts now display as an exclamation mark nested inside a yellow circle.
+* Error alerts now display as an exclamation mark nested inside a red triangle.
+* Success alerts now display as a green tick.
+
+For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].
+
+=== Advanced Code Editor 3.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.1.0 includes the following improvement.
+
+
+==== new option to display the Advanced Code Editor inline instead of in a dialog
+
+In {productname} 6.3, the Advanced Code Editor plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within TinyMCE’s existing editor space instead of been displayed in a seperate dialog box.
```

Comment:
```suggestion
In {productname} 6.3, the Advanced Code Editor plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within TinyMCE’s existing editor space instead of being displayed in a separate dialog box.
```